Documentation/project-docs/conferences.md

```diff
@@ -0,0 +1,47 @@
+## .NET Conferences and Announcements links
+
+### .NET Core 3.0
+
+* [.NET Core 3.0 previews](https://github.com/dotnet/core/tree/master/release-notes/3.0)
+    * [.NET Core 3.0 Preview 5](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-preview-5/) (on Mon 5/5)
+    * RC (Release Candidate) - July 2019 (see [.NET 5 announcement blog](https://devblogs.microsoft.com/dotnet/introducing-net-5/))
+    * GA (General Availability) - September 2019 (see [.NET 5 announcement blog](https://devblogs.microsoft.com/dotnet/introducing-net-5/))
+* [.NET Core 3 and Support for Windows Desktop Apps](https://blogs.msdn.microsoft.com/dotnet/2018/05/07/net-core-3-and-support-for-windows-desktop-applications/) - WPF, WinForms, etc.
+
+### .NET plans
+
+* .NET Core 3.1 - LTS - November 2019 (see [.NET 5 announcement blog](https://devblogs.microsoft.com/dotnet/introducing-net-5/))
+* [.NET 5](https://devblogs.microsoft.com/dotnet/introducing-net-5/) - November 2020 GA
```

Comment:
This is redundant with roadmap on https://github.com/dotnet/core. I do not think we need to be maintaining a duplicate information here.

Reply:
I agree, I think if we need such a document as this it should probably be in dotnet/core. I'd dump it.